clientGQL_test.go

```diff
@@ -83,6 +140,11 @@ func RegisterPaginatedEndpoint(t *testing.T, endpoint string, requests ...TestRe
 	return autopilot.Server.URL + url
 }
 
+func TmpBetterTestClient(t *testing.T, endpoint string, testRequest TestRequest) *ol.Client {
+	oldStyleRequest := fmt.Sprintf(`{%s, %s}`, testRequest.Request, testRequest.Variables)
```

Comment:
we shouldn't do this - this is likely making things WAY more complicated.

We should make the new `TheBestTestClient` which natively works with `[]TestRequest`

I have a pretty clear picture in my head of how that will work - do you want me to mock it up for you?

Reply:
To be updated per offline conversation